gems/activerecord/6.0/activerecord.rbs

```diff
@@ -40,6 +40,7 @@ module ActiveRecord
 
     def self.abstract_class=: (bool) -> void
     def self.scope: (Symbol, ^(*untyped) -> untyped ) -> void
+                  | (Symbol, Proc) -> void
```

Comment:
I talked with @pocke and @mame about this change. And the conclusion is writing `Proc` here is the correct type, not `^(*untyped) -> untyped`. So, I'd ask you to remove the first overload too.

The `^(*untyped) -> untyped` means that the proc passed here should accept any number of arguments.

```
scope :foo, -> (*args) { }           # This is ok
scope :bar, -> (x) { }               # This is a type error because the passed proc only accepts exactly one argument
```

(If we try above code with Steep, it accepts both lines. This is a bug of Steep.)